src/common/user-store/user-store.ts

```diff
@@ -181,7 +192,12 @@ export class UserStore extends BaseStore<UserStoreModel> /* implements UserStore
 
     return toJS({
       lastSeenAppVersion: this.lastSeenAppVersion,
-      preferences,
+
+      preferences: {
+        ...preferences,
+
+        updateChannel: this.dependencies.selectedUpdateChannel.value.get().id,
```

Comment:
Why should this be here? Is there something wrong with having the user (preferences) store itself be the single source of truth?

Reply:
Because it we shouldn't create god classes like `UserStore` at all. E.g. if I want to inject `allowErrorReporting`, I shouldn't have to get knowledge of used `colorTheme`. Splitting these to smaller pieces gives us possibility to compose them without fear of circular dependencies and minimal cognitive load.

Reply:
Sure but the direction of the injectables could easily be in the other direction. (`userStoreInjectionToken` -> `allowErrorReporting`)

Reply:
It would only lead to one more place that uses the god class.

Reply:
Resolved for later possible conversation elsewhere.